public/app/features/explore/Wrapper.tsx

```diff
@@ -87,22 +119,56 @@ class WrapperUnconnected extends PureComponent<Props> {
     document.title = documentTitle;
   }
 
+  updateSplitSize(rightPaneWidth: number) {
```

Comment:
kinda nit for readability: can we make this an arrow function so that on line 156 we can do
```
<SplitView [...] onResize={this.updateSplitSize}>[...]
```